sources/System/BombHolder/BombHolderSystem.hpp

```diff
@@ -22,13 +32,33 @@ namespace BBM
 		void _spawnBomb(Vector3f position, BombHolderComponent &holder, unsigned id);
 
 		//! @brief Spawn a bomb at the specified position.
-		static void _dispatchExplosion(Vector3f position, WAL::Wal &, int count);
+		static void _dispatchExplosion(const Vector3f &position, WAL::Wal &wal, int radiusToDo,
+		                               ExpansionDirection expansionDirections);
+
+		//! @brief Wrapped call to specify default arg value
+		inline static void _dispatchExplosion(const Vector3f &position, WAL::Wal &wal, int radiusToDo)
```

Comment:
Please specify this when you call the method instead of doing an inline function in a header to specify this. This feels like a hack and is not really readable.